components/preview/README.md

```diff
@@ -0,0 +1,51 @@
+# @hpcc-js/wc-preview
+
+**@hpcc-js/wc-preview** is a Web Component library for embedding live html / JavaScript snippets in your website, ideal for hosting demos in documentation.
+
+* [Installing](#installing)
+* [Preview](src/preview)
+
+## Installing
+
+For use with Webpack, Rollup, or other Node-based bundlers, @hpcc-js/wc-preview is typically installed via a package manager such as Yarn or npm. (@hpcc-js/wc-preview is distributed primarily as an ES module.)
+
+```bash
+npm install --save @hpcc-js/wc-preview
+```
+
+@hpcc-js/wc-preview can then be imported as a namespace:
+
+```js
+import * as preview from "@hpcc-js/wc-preview";
+```
+
+In vanilla HTML, @hpcc-js/wc-preview can be imported as an ES module, say from Skypack:
+
+```html
+<script type="module">
+
+import * as preview from "https://cdn.skypack.dev/@hpcc-js/wc-preview";
+
+document.body.innerHTML("<hpcc-preview>...</hpcc-preview>");
+
+</script>
+
+<hpcc-preview>
+...
+</hpcc-preview>
+
+```
+
+@hpcc-js/wc-preview is also available as a UMD bundle for legacy browsers.
+
+```html
+<head>
+    <script src="https://cdn.jsdelivr.net/npm/@hpcc-js/wc-preview"></script>
+</head>
+
+<body>
+    <hpcc-preview>
+        ...
+    </hpcc-preview>
+</body>
```

Comment:
Maybe these issues will be resolved when these changes are published?

The ES Module preview throws this error: Uncaught Error: [Package Error] "tslib" no dependency version info found. (Imported by "@hpcc-js/wc-core").

The UMD preview throws a 404 error for this file: "assets/index.umd.min.js"

Reply:
The work on hpcc-preview is just starting.